src/aws_sigv4_internal.hrl

```diff
@@ -1,12 +1,10 @@
 -ifndef(_AWS_SIGV4_INTERNAL_HRL_).
 -define(_AWS_SIGV4_INTERNAL_HRL_, true).
 
--type headers() :: [{binary(), binary()}].
-
 -record(request,
         { method :: binary()
         , url :: binary()
-        , headers :: headers()
+        , headers :: aws_sigv4_internal:headers()
```

Comment:
Type is not exported (unless I'm blind on my phone)

Reply:
They are in the `-export_type` block in `aws_sigv4_internal.erl`.